### DISCUSSION TITLE
change media_library_original When using (withResponsiveImages) method

### DISCUSSION THREAD

**Discussion by ghost** [spatie/laravel-medialibrary#2699] [Q&A] [answered]
Hi, I am using the method withResponsiveImages to create resonant images.
But I have a problem, there is the word "media_library_original" in the name of the created resonant files and I want to personalize it. Please help me.

**Comment by erikn69** [comment, marked as answer]
Maybe it's not possible
https://github.com/spatie/laravel-medialibrary/search?q=media_library_original
Looks like a lot overwrite for make that
